examples/demo/gloo/isocurve_for_trisurface_qt.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""
+This example demonstrates isocurve for triangular mesh with vertex datas and a
+ qt interface.
+"""
+
+import sys
+import numpy as np
+
+from vispy import scene
+
+from vispy.geometry.generation import create_sphere
+from vispy.color.colormap import get_colormaps
+
+try:
+    from sip import setapi
+    setapi("QVariant", 2)
+    setapi("QString", 2)
+except ImportError:
+    pass
+
+from PyQt4 import QtGui, QtCore
+
+
+class ObjectWidget(QtGui.QWidget):
+    """
+    Widget for editing OBJECT parameters
+    """
+    signal_objet_changed = QtCore.pyqtSignal(name='objectChanged')
+
+    def __init__(self, parent=None):
+        super(ObjectWidget, self).__init__(parent)
+
+        l_nbr_steps = QtGui.QLabel("Nbr Steps ")
+        self.nbr_steps = QtGui.QSpinBox()
+        self.nbr_steps.setMinimum(3)
+        self.nbr_steps.setMaximum(100)
+        self.nbr_steps.setValue(6)
+        self.nbr_steps.valueChanged.connect(self.update_param)
+
+        l_cmap = QtGui.QLabel("Cmap ")
+        self.cmap = list(get_colormaps().keys())
+        self.combo = QtGui.QComboBox(self)
+        self.combo.addItems(self.cmap)
+        self.combo.currentIndexChanged.connect(self.update_param)
+
+        gbox = QtGui.QGridLayout()
+        gbox.addWidget(l_cmap, 0, 0)
+        gbox.addWidget(self.combo, 0, 1)
+        gbox.addWidget(l_nbr_steps, 1, 0)
+        gbox.addWidget(self.nbr_steps, 1, 1)
+
+        vbox = QtGui.QVBoxLayout()
+        vbox.addLayout(gbox)
+        vbox.addStretch(1.0)
+
+        self.setLayout(vbox)
+
+    def update_param(self, option):
+        self.signal_objet_changed.emit()
+
+
+class MainWindow(QtGui.QMainWindow):
+
+    def __init__(self):
+        QtGui.QMainWindow.__init__(self)
+
+        self.resize(700, 500)
+        self.setWindowTitle('vispy example ...')
+
+        splitter = QtGui.QSplitter(QtCore.Qt.Horizontal)
+
+        self.canvas = Canvas()
+        self.canvas.create_native()
+        self.canvas.native.setParent(self)
+
+        self.props = ObjectWidget()
+        splitter.addWidget(self.props)
+        splitter.addWidget(self.canvas.native)
+
+        self.setCentralWidget(splitter)
+        self.props.signal_objet_changed.connect(self.update_view)
+        self.update_view()
+
+    def update_view(self):
+        # banded, nbr_steps, cmap
+        self.canvas.set_data(self.props.nbr_steps.value(),
+                             self.props.combo.currentText())
+
+
+class Canvas(scene.SceneCanvas):
```

Comment:
Since this is a `SceneCanvas` and not an `app.Canvas` and doesn't use `gloo` directly at all, this example should probably go in `examples/demo/scene/` instead of `examples/demo/gloo`